### PR TITLE
speed up assign_release_to_builds

### PR DESCRIPTION
**Pull request by xsuchy**
if we do not commit after each package it is ~1000x faster

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

**Comment by marusak** [comment]
ok to test

**Comment by marusak** [comment]
add to whitelist

**Comment by marusak** [comment]
@trapas test this please

**Comment by marusak** [comment]
test this please

File: src/pyfaf/actions/assign_release_to_builds.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def run(self, cmdline, db):
                 if found_build:
                     self._add_into_build_opsysrelease_arch(db, found_build,
                                                            opsysrelease, arch)
+                db.session.flush()
```
Comment (marusak) [inline src/pyfaf/actions/assign_release_to_builds.py]: Shouldn't this be one level up? It is in the for loop, wasn't the point of this commit to have flush outside loops?

**Comment by jfilak** [comment]
What about to call flush after 1000 iterations? I'm not sure it is safe to commit huge data. Do you know why we commit data after each loop?

File: src/pyfaf/actions/assign_release_to_builds.py (hunk 1 above)
Comment (xsuchy) [inline src/pyfaf/actions/assign_release_to_builds.py]: It should. I will change it.

**Comment by xsuchy** [comment]
@jfilak Saving data after each loop have sense if those data are expensive to extract, so you have safepoint, from which you can restart.
However these data are easy to extract. The original code run over night and quick interpolation was that it may run for another 45 days. More or less. After this change, the operation was finished after lunch break.
And yes flush after 1000 iteration can be done too. But that commit is not huge. Each for-loop consist only from several thousands of rows. And the journal is not big (just few MB) and easily fit in memory.

**Comment by marusak** [comment]
I've updated this PR so it flushes after 1000 iterations or before the action finishes.

**Comment by marusak** [comment]
ok to test

**Pull request merged**